client/js/lounge.js

```diff
@@ -140,7 +140,9 @@ $(function() {
 		$("#sign-in").remove();
 
 		var id = data.active;
-		var target = sidebar.find("[data-id='" + id + "']").trigger("click");
+		var target = sidebar.find("[data-id='" + id + "']").trigger("click", {
+			replace: true
```

Comment:
Nitpick, could you rename this into `replaceState`, or `replaceHistory`, or something like that? `replace` is too generic for my taste (we could need it later, but even now it's hard to figure out what it means without reading your code below since event handling is so "detached" compared to normal function calls).